src/EventListener/AddLibrarySubscriber.php

```diff
@@ -58,20 +57,15 @@ public function add(GetResponseEvent $event): void
             return;
         }
 
-        $this->tagBag->add(new TwigTag(
-            '@SetonoSyliusCriteoPlugin/Tag/library.html.twig',
-            TagInterface::TYPE_HTML,
-            Tags::TAG_LIBRARY
-        ), TagBagInterface::SECTION_HEAD);
+        $tag = new TemplateTag('@SetonoSyliusCriteoPlugin/Tag/library.html.twig');
+        $tag->setSection(TagInterface::SECTION_HEAD);
+        $this->tagBag->addTag($tag);
 
-        $this->tagBag->add(new TwigTag(
-            '@SetonoSyliusCriteoPlugin/Tag/default_events.js.twig',
-            TagInterface::TYPE_SCRIPT,
-            Tags::TAG_DEFAULT_EVENTS,
-            [
-                'account_id' => $this->getAccount()->getAccountId(),
-                'site_type' => $this->siteTypeResolver->resolve(),
-            ]
-        ), TagBagInterface::SECTION_BODY_END);
+        $tag = new TemplateTag('@SetonoSyliusCriteoPlugin/Tag/default_events.js.twig', [
```

Comment:
Use twig tag instead (require `setono/tag-bag-twig`) and refactor the template to `default_events.html.twig`